files/spilo/postgres-appliance/scripts/configure_spilo.py

```diff
@@ -310,7 +310,7 @@ def deep_update(a, b):
     pg_stat_statements.track_utility: 'off'
     extwlist.extensions: 'btree_gin,btree_gist,citext,extra_window_functions,first_last_agg,hll,\
 hstore,hypopg,intarray,ltree,pgcrypto,pgq,pgq_node,pg_ivm,pg_trgm,postgres_fdw,mysql_fdw,multicorn,\
-parquet_s3_fdw,tablefunc,uuid-ossp'
+parquet_s3_fdw,vector,tablefunc,uuid-ossp'
```

Comment:
fine for now, but we need to think about how to get out of this business of adding stuff to this setting